src/components/formInput/view/formInputView.js

```diff
@@ -13,123 +13,107 @@ import { getResizedAvatar } from '../../../utils/image';
 // Styles
 import styles from './formInputStyles';
 
-class FormInputView extends Component {
-  /* Props
-   * ------------------------------------------------
-   *   @prop { string }     placeholder       - Place holder text.
-   *   @prop { string }     type              - Input type.
-   *   @prop { boolean }    isFirstImage      - Render image from steem.
-   *   @prop { boolean }    isEditable        - Can permission edit.
-   *   @prop { boolean }    isValid           - This delegate input valit or not.
-   *   @prop { boolean }    secureTextEntry   - For hiding password value.
-   */
-  constructor(props) {
-    super(props);
+const FormInputView = ({
+  placeholder,
+  type,
+  isFirstImage,
+  isEditable,
+  leftIconName,
+  rightIconName,
+  secureTextEntry,
+  iconType,
+  wrapperStyle,
+  height,
+  inputStyle,
+  onChange,
+  isValid,
+  value,
+}) => {
+  const [_value, setValue] = useState(value || '');
+  const [inputBorderColor, setInputBorderColor] = useState('#e7e7e7');
+  const [_isValid, setIsValid] = useState(true);
 
-    this.state = {
-      value: props.value || '',
-      inputBorderColor: '#e7e7e7',
-      isValid: true,
-    };
-  }
+  const _handleOnChange = text => {
+    setValue(text);
 
-  // Component Functions
-  _handleOnChange = value => {
-    const { onChange } = this.props;
-
-    this.setState({ value });
     if (onChange) {
-      onChange(value);
+      onChange(text);
     }
   };
 
-  _handleOnFocus = () => {
-    this.setState({ inputBorderColor: '#357ce6' });
+  const _handleOnFocus = () => {
+    setInputBorderColor('#357ce6');
   };
 
-  // Component Life Cycles
-  UNSAFE_componentWillReceiveProps(nextProps) {
-    const { isValid } = this.props;
-
-    if (nextProps.isValid !== isValid) {
-      this.setState({ isValid: nextProps.isValid });
-    }
-  }
+  // Workaround for android context (copy/paste) menu
+  useEffect(() => {
+    setValue(' ');
+    setTimeout(() => {
```

Comment:
i guess this is required for context menu to resolve or something 🤔

Reply:
it is a workaround solution

Reply:
Btw, Do we have this problem in Editor ?

Reply:
it is strange because we solved to this problem on editor with `contextMenuHidden={false}` code, but it is not working on here